api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -1176,7 +1176,7 @@ private EncounterWithCodedObsCohortDefinition getStateOfStayCohort() {
    *
    * @return CohortDefinition
    */
-  private DateObsCohortDefinition getPatientsWithMasterCardDrugPickUpDate() {
+  public DateObsCohortDefinition getPatientsWithMasterCardDrugPickUpDate() {
```

Comment:
why visibility  of this    method was changed to  public?

Reply:
It happened probably by mistake! im going to check